wordpress-mobile/gutenberg-mobile
# Fix for deleting all text on the More block resets it to the default text

This PR fixes #359 by making sure to only use the default text once, during init of the block.
(The web side does the same).

GB side companion PR: https://github.com/WordPress/gutenberg/pull/12821

Steps to test:
- Tap on a Read More block
- Delete all the text
- It should stay empty

Test 2:
- Add a new Read More block
- The text should be "Read more"
- Delete all the text
- It should stay empty
